app/src/main/java/to/bitkit/repositories/LightningRepo.kt

```diff
@@ -222,7 +229,7 @@ class LightningRepo @Inject constructor(
 
             Result.success(Unit)
         } catch (e: Throwable) {
-            if (shouldRetry) {
+            if (shouldRetry && !isRecoveryMode) {
```

Comment:
nit: can't get to this code path when `isRecoveryMode` is true, ` && !isRecoveryMode` check is redundant and safely removable.